app/client/src/components/shared/WaterbodyInfo.tsx

```diff
@@ -928,11 +928,16 @@ function WaterbodyInfo({
   const tribeContent = labelValue('Tribe Name', attributes.TRIBE_NAME);
 
   // jsx
-  const upstreamWatershedContent = (
+  const watershedContent = (
     <>
       {labelValue(
         'Area',
-        attributes.areasqkm && `${formatNumber(attributes.areasqkm)} sq. km.`,
+        attributes.areasqkm &&
+          attributes.areaacres &&
+          `${formatNumber(attributes.areaacres)} acres / ${formatNumber(
+            attributes.areasqkm,
+            2,
+          )} km²`,
```

Comment:
```suggestion
        (attributes.areaacres
          ? `${formatNumber(attributes.areaacres)} acres`
          : '') +
          (attributes.areasqkm && attributes.areaacres ? ' / ' : '') +
          (attributes.areasqkm
            ? `${formatNumber(attributes.areasqkm, 2)} km²`
            : ''),
```

The upstream watershed popup no longer displays anything for the area. Unfortunately, that layer doesn't have the `areaacres` attribute. We should try to bring this up at tomorrow's meeting as well. Maybe the upstream watershed service could be updated to return acres. 

The above suggestion will get it back to what it was before.

We could also try converting sqkm to acres, like below.

```suggestion
        attributes.areasqkm &&
          `${formatNumber(
            attributes.areaacres
              ? attributes.areaacres
              : attributes.areasqkm / 0.00404686,
          )} acres / ${formatNumber(attributes.areasqkm, 2)} km²`,
```

Reply:
Thanks, I forgot that the square acres aren't in that service. I think calculating the value is fine.